contracts/external/cw-fund-distributor/src/contract.rs

```diff
@@ -127,19 +132,20 @@ pub fn execute_fund_native(
     let mut attributes: Vec<(String, String)> = Vec::new();
```

Comment:
i really like how you collect the attributes here. it's a small thing, but potentially really helpful in the future.